### DISCUSSION TITLE
내가 Next.js 블로그를 시작한 배경 | thisyujeong.dev

### DISCUSSION THREAD

**Discussion by giscus** [thisyujeong/thisyujeong-dev#15] [General]
# 내가 Next.js 블로그를 시작한 배경 | thisyujeong.dev

Next.js로 정적블로그를 만들게 된 과정을 풀어봅니다.

https://thisyujeong.dev/blog/created-process

**Comment by junghan0611** [comment]
Hugo 블로그를 쓰는 데 욕심이 나네요. 좋은 정보 감사드립니다.

**Comment by vlogcloud** [comment]
글 잘보고갑니다 :) 요새는 블로그 안하시나요~?

**Reply by thisyujeong** [reply]
안녕하세요, lifethehunt님! 댓글 감사합니다!
여전히 블로그 운영하고 있습니다 :) 학업으로 바빠서 요새 글이 뜸했어요. 블로그 Next.js 14버전으로 마이그레이션을 진행하고 있어요! 다음 글은 마이그레이션 포스트로 찾아올 것 같으니 또 방문해주세요 ㅎㅎ

**Reply by vlogcloud** [reply]
답장 감사해요 👍👏👏 좋은 포스팅들 기대하고 있을게요 ☺️ 

[블로그](https://lifethehunt.com/)

제 블로그도 남겨놓고 팔로우하고 갑니당~🫡🫡

기술블로그는 아니에욧! 😗😗

**Reply by thisyujeong** [reply]
감사합니다😊 저도 팔로우하고 종종 방문할게요 ㅎㅎ
